### DISCUSSION TITLE
Dynamic computing resources

### DISCUSSION THREAD

**Discussion by Zruto** [nextflow-io/nextflow#4118] [Q&A] [answered]
Hello, I was wondering if there is any parameter to get the running memory of the current task when retrying, and then increase it instead of specifying the value yourself, because the memory required by each task is not the same.
![image](https://github.com/nextflow-io/nextflow/assets/46238825/35d9efe9-8df4-4839-a8a2-d6219e850534)

**Comment by pditommaso** [comment, marked as answer]
You can use `task.memory` to get memory requested in the failed attempt
